components/brave_rewards/resources/extension/brave_rewards/utils.ts

```diff
@@ -168,3 +172,9 @@ export const onVerifyClick = (actions: any, externalWallet?: RewardsExtension.Ex
 
   handleUpholdLink(externalWallet.verifyUrl)
 }
+
+export const getClaimedPromotions = (promotions: RewardsExtension.Promotion[]) => {
+  return promotions.filter((promotion: RewardsExtension.Promotion) => {
+    return promotion.status === 4 // PromotionStatus::FINISHED
```

Comment:
nit: TODO so a const doesn't get forgotten

Reply:
right now we can't use direct definition so I added reference for a number